src/main/webapp/navigation.jsp

```diff
@@ -0,0 +1,7 @@
+<nav>
```

Comment:
Can you try moving the `navigation.jsp` file to be inside the `WEB-INF` directory, that way users can't access it directly?

Example here: https://stackoverflow.com/a/20868516/873165

(If this doesn't work then let's not worry too much about it, it's not really a big deal.)